File: app/src/main/java/workshop/akbolatss/tagsnews/screen/reminders/RemindersView.java
package workshop.akbolatss.tagsnews.screen.reminders;

import java.util.List;

import workshop.akbolatss.tagsnews.base.BaseView;
import workshop.akbolatss.tagsnews.model.dao.ReminderItem;

public interface RemindersView extends BaseView {

    void onUpdateReminders();

    void onAddReminder();

    void onShowReminders(List<ReminderItem> reminderItems);
}

File: app/src/main/java/workshop/akbolatss/tagsnews/screen/reminders/RemindersView.kt
package workshop.akbolatss.tagsnews.screen.reminders

import workshop.akbolatss.tagsnews.base.BaseView
import workshop.akbolatss.tagsnews.model.dao.ReminderItem

/**
 * MVP View for #RemindersView
 * @see RemindersActivity
 */
interface RemindersView : BaseView {

    fun onNoContent(isEmpty: Boolean)

    fun onAddReminder()

    fun onShowReminders(reminderItems: List<ReminderItem>)
}
